guides/source/security.md

```diff
@@ -659,19 +659,31 @@ Also, the second query renames some columns with the AS statement so that the we
 
 Ruby on Rails has a built-in filter for special SQL characters, which will escape `'` , `"` , NULL character, and line breaks. *Using `Model.find(id)` or `Model.find_by_some thing(something)` automatically applies this countermeasure*. But in SQL fragments, especially *in conditions fragments (`where("...")`), the `connection.execute()` or `Model.find_by_sql()` methods, it has to be applied manually*.
 
-Instead of passing a string to the conditions option, you can pass an array to sanitize tainted strings like this:
```

Comment:
I had hard time figuring out what the `conditions option` meant, seeing the history made more sense https://github.com/rails/rails/commit/4d4819fb:

```diff
Instead of passing a string to the conditions option, you can pass an array to sanitize tainted strings like this:

<ruby>
-Model.first(:conditions => ["login = ? AND password = ?", entered_user_name, entered_password])
+Model.where("login = ? AND password = ?", entered_user_name, entered_password).first
</ruby>

```

Since I start using Rails on V4, that old way was not there anymore.